### DISCUSSION TITLE
How to remove screensaver

### DISCUSSION THREAD

**Discussion by 13gears13** [rocksdanister/lively#1981] [General]
**Describe the bug**
Game freezes, NOT RESPONDING, necessitating END TASK

**To Reproduce**
Install the Lively.scr 

**Desktop (please complete the following information):**
Win10x64

**Additional context**
I uninstalled LIVELYWALLPAPER, but the Lively.scr remains in the system, so now I don't know how to uninstall it and I'm hooped.  Someone help.  The screensaver now glitches when it attempts to come on.  I'll try to disable screen saver in the meantime.  But I'd like a method of how to remove the Lively.scr keys from registry fully and completely.  Thanks.

**Log file (Important)**
deleted not found

**Comment by rocksdanister** [comment]
1. Select a different screensaver in [Windows settings](https://www.windowscentral.com/how-enable-screen-savers-windows-10)
2. [Delete the .scr file from C:\Windows](https://github.com/rocksdanister/lively/wiki/Screen-Saver#1-windows-screensaver-file) 

The .scr is just a simple console app that calls Lively and it should not result in game crashing.

**Reply by 13gears13** [reply]
Hi.  I don't believe the .scr file is causing the game freezing/crashing.  What is happening is that the Lively Wallpaper was set to run as a screen saver but I uninstalled Lively and the option within the screen saver settings was still set to Lively.  So the uninstaller failed to remove the screensaver designation.  All I want to do now is remove the registry entry which was created by the Lively.scr command.  Anyone know where it planted itself please comment.  thanks.

**Reply by rocksdanister** [reply]
That's what I am saying, screensaver file does not open Lively Wallpaper - it just tries to call it if available otherwise skips.

Deleting the file is all that is required, Windows will handle the rest.
https://www.screensaversplanet.com/help/guides/windows/how-to-remove-a-screensaver-on-windows-7-54

>By manually removing a screensaver file from those folders, you are effectively uninstalling the screensaver, and it will no longer be listed among your other screensavers in the Screen Saver Settings panel.
